python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -134,11 +133,13 @@ class GaudiExec(IPrepareApp):
         'platform' :    SimpleItem(defvalue='x86_64-slc6-gcc49-opt', typelist=[str], doc='Platform the application was built for'),
         'extraOpts':    SimpleItem(defvalue='', typelist=[str], doc='An additional string which is to be added to \'options\' when submitting the job'),
         'extraArgs':    SimpleItem(defvalue=[], typelist=[str], sequence=1, doc='Extra runtime arguments which are passed to the code running on the WN'),
+        'getMetadata':  SimpleItem(defvalue=True, doc='Do you want to get the metadata from your jobs'),
```

Comment:
Can we add this to the docstr maybe with an example of it's use and what the users can expect when they set this to True.
(I'd also try setting this false by default to not change the default way of working for people as this may have unintentional effects)